cloudbuild-test.yaml

```diff
@@ -18,8 +18,10 @@ steps:
   id: Replace
   script: |
     #!/usr/bin/env bash
-    sed -i "s/%GCLOUD_PROJECT%/${PROJECT_ID}/g" k8s/base/*
+    grep -rlZ GCLOUD_PROJECT k8s/base/. | xargs -0 sed -i "s/\${GCLOUD_PROJECT}/$PROJECT_ID/g"
+    grep -rlZ GCP_REGION k8s/base/. | xargs -0 sed -i "s/\${GCP_REGION}/$LOCATION/g"
```

Comment:
Do we need instructions to set `LOCATION` as well ?

Reply:
This is just for the cloud build test.  The LOCATION env var and PROJECT_ID env vars are set by default in cloud build: https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_default_substitutions

Users just need to set GCP_REGION and GCLOUD_PROJECT.  Users use envsubst instead of sed because it is simpler.  The tests use sed because envsubst can't write to the original files very easily.

Reply:
Makes sense. It wasn't very clear from where those two variables were coming from.

Might be good to put a small comment informing users to check the default substitutions made by cloud build.

Reply:
updated the comment.